collections/hub/golden_path/feature_tests/block_transfer.json

```diff
@@ -7,6 +7,629 @@
+              {
+                "id": 1,
+                "description": "Status code is 202",
+                "exec": [
+                  "expect(response.status).to.equal(202)"
+                ]
+              },
+              {
+                "id": 2,
+                "description": "Payee FSP 'fspiop-source' is 'testingtoolkitdfsp'",
+                "exec": [
+                  "if (environment.ENABLE_WS_ASSERTIONS===true) {",
+                  "expect(requestVariables.payeeRequest.headers['fspiop-source']).to.equal(environment.fromFspId)",
+                  "}"
+                ]
+              },
+              {
+                "id": 3,
+                "description": "Payee FSP 'fspiop-destination' is 'payeefsp'",
+                "exec": [
+                  "if (environment.ENABLE_WS_ASSERTIONS===true) {",
+                  " expect(requestVariables.payeeRequest.headers['fspiop-destination']).to.equal(environment.SIMPAYEE_NAME)",
+                  "}"
+                ]
+              },
+              {
+                "id": 4,
+                "description": "Payee FSP: 'fspiop-signature' is same as sent in the request",
+                "exec": [
+                  "if (environment.ENABLE_WS_ASSERTIONS===true) {",
+                  " expect(requestVariables.payeeRequest.headers['fspiop-signature']).to.equal(request.headers['fspiop-signature'])",
+                  "}",
+                  ""
+                ]
+              },
+              {
+                "id": 5,
+                "description": "Payee FSP 'accept' is same as sent in the request",
+                "exec": [
+                  "if (environment.ENABLE_WS_ASSERTIONS===true) {",
+                  " expect(requestVariables.payeeRequest.headers['accept']).to.equal(request.headers['Accept'])",
+                  "}"
+                ]
+              },
+              {
+                "id": 6,
+                "description": "Payee FSP 'content-type' is same as sent in the request",
+                "exec": [
+                  "if (environment.ENABLE_WS_ASSERTIONS===true) {",
+                  " expect(requestVariables.payeeRequest.headers['content-type']).to.equal(request.headers['Content-Type'])",
+                  "}"
+                ]
+              },
+              {
+                "id": 7,
+                "description": "Payee FSP 'fspiop-uri' is /quotes",
+                "exec": [
+                  "if (environment.ENABLE_WS_ASSERTIONS===true) {",
+                  " expect(requestVariables.payeeRequest.headers['fspiop-uri']).to.equal(\"/quotes\")",
+                  "}"
+                ]
+              },
+              {
+                "id": 8,
+                "description": "Payee FSP 'fspiop-http-method' is 'POST'",
+                "exec": [
+                  "if (environment.ENABLE_WS_ASSERTIONS===true) {",
+                  " expect(requestVariables.payeeRequest.headers['fspiop-http-method']).to.equal(\"POST\")",
```

Comment:
@vijayg10 ... the description refers to Payee FSP, but you are referring to requestVariables. Is it Payer or Payee?

Reply:
Done